app/services/create_batch.rb

```diff
@@ -2,6 +2,11 @@ class CreateBatch
   def call
     unprocessed_appointment_summaries = AppointmentSummary.unprocessed
 
+    # exclude braille until the print house can actually deal with it
+    unprocessed_appointment_summaries = unprocessed_appointment_summaries.reject do |as|
```

Comment:
@stevenwilkin and I discussed this a bit, and it would add more moving parts for not much benefit on something which is short-lived. If we start blowing memory limits on dynos than obviously that changes things, and we'd have to revisit this.